clusterloader2/pkg/measurement/common/simple/profile.go

```diff
@@ -44,6 +44,9 @@ type memoryProfileMeasurement struct{}
 
 // Execute gathers memory profile of a given component.
 func (c *memoryProfileMeasurement) Execute(config *measurement.MeasurementConfig) ([]measurement.Summary, error) {
+	// TODO: Introduce two actions:
```

Comment:
NIT: Ownerless/bugless TODO, here and in other places.

Reply:
I'm working on it as we speak - that's why I didn't add it.